frontend/components/WaitingPages/HostWaiting.tsx

```diff
@@ -56,115 +88,143 @@ export default function HostWaiting({ hostId }: hostProps) {
 		displayPlayers.push(row);
 	}
 
-	const headerContent = (
-		<div className="flex flex-col p-4">
-			<div className="text-4xl p-2 xl:text-5xl">Options:</div>
-			<div className="flex flex-row">
-				<div className="flex flex-row p-1">
-					<div className="flex flex-col text-xl xl:text-2xl whitespace-nowrap">
-						<div className="mb-2">Time Limit:</div>
-						<div className="mb-2">Max Players:</div>
+	const qrCode = qr(gameId);
+
+	return (
+		<main
+			className={`${inter.className} w-screen h-screen font-sans bg-purple-100 flex flex-col justify-between p-4`}
+		>
+			<div className="flex flex-row items-center justify-center sm:justify-start">
+				<Image
+					alt="Kakaw logo"
+					className="mx-4 hidden sm:block"
+					src={logo2}
+					width={220}
+					style={{
+						maxWidth: '100%',
+						height: 'auto',
+					}}
+				/>
+				<div className="bg-gray-100 flex flex-col sm:flex-row items-center justify-center sm:justify-between font-extrabold shadow-heavy rounded-xl p-3 sm:w-max">
+					<div className="sm:mx-4">{qrCode}</div>
+					<div className="flex flex-col items-center justify-center sm:mx-4">
+						<div className="text-4xl 2xl:text-5xl whitespace-nowrap">
+							Join with the code:
+						</div>
+						<div className="text-8xl 2xl:text-9xl">{gameId}</div>
+						<div className="text-2xl 2xl:text-3xl whitespace-nowrap">
+							at https://www.kakaw.com
```

Comment:
Let's make this and the localhost:3000 in qrCode.tsx line 8 configurable in the same place